paf-mvp-core-js/src/monitoring/tracer-factory.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  trace,
+  Tracer,
+  Span,
+  SpanOptions,
+  SpanContext,
+  SpanStatus,
+  Attributes as SpanAttributes,
+  AttributeValue as SpanAttributeValue,
+  TimeInput,
+  Context,
+  Exception,
+} from '@opentelemetry/api';
+
+class NoopSpanContext implements SpanContext {
+  spanId: string;
+  traceFlags: number;
+  traceId: string;
+}
+
+class NoopSpan implements Span {
```

Comment:
Please add comments about why there is a Noop implementation.

Reply:
The Tracer is the contract interface for OpenTelemetry. The user doesn't need to access or know anything about the implementation. Noop is the implementation of "do nothing".